va/va_trace.c

```diff
@@ -5867,3 +5867,26 @@ void va_TraceStatus(VADisplay dpy, const char * funcName, VAStatus status)
     va_TraceMsg(trace_ctx, "=========%s ret = %s, %s \n",funcName, vaStatusStr(status), vaErrorStr(status));
     DPY2TRACE_VIRCTX_EXIT(pva_trace);
 }
+
+void va_TraceImageInfo(VADisplay dpy, VAImage *image)
+{
+    int i;
+
+    DPY2TRACE_VIRCTX(dpy);
+
+    TRACE_FUNCNAME(idx);
+
```

Comment:
We probably need to check that image is not null. Won't be good to crash during tracing...